redbot/core/commands/requires.py

```diff
@@ -583,7 +583,10 @@ def _get_rule_from_ctx(self, ctx: "Context") -> PermState:
         if author.voice is not None:
             channels.append(author.voice.channel)
         channels.append(ctx.channel)
```

Comment:
Probably makes sense to check for permissions in the parent channel here?
```py
        channels.append(ctx.channel.parent)
```
Or maybe we should check for both in case someone comes up with an idea of keeping the thread active for long time and wanting *different* permissions in it from the parent for some reason?
```py
        channels.append(ctx.channel.parent)
        channels.append(ctx.channel)
```

// I purposefully did not use code suggestion here - you should probably put the appropriate (whichever you think makes more sense) code in the existing `if isinstance`.